library/core/src/main/java/com/google/android/exoplayer2/video/MediaCodecVideoRenderer.java

```diff
@@ -679,16 +679,16 @@ private void clearRenderedFirstFrame() {
     }
   }
 
-  /* package */ void maybeNotifyRenderedFirstFrame() {
+  /* package */ void maybeNotifyRenderedFirstFrame(long framePosition) {
```

Comment:
It's probably going to less error prone if this method and the one below receive `framePositionUs` and divide by 1000 internally (rather than having the callers need to divide).